core/src/main/java/org/apache/calcite/plan/AbstractRelOptPlanner.java

```diff
@@ -202,6 +204,11 @@ public boolean isRuleExcluded(RelOptRule rule) {
     return this;
   }
 
+  @Experimental
```

Comment:
I think we have switched to using apiguardian for this kind of annoations. 
`@API(since = "1.28", status = API.Status.EXPERIMENTAL)`

Moreover, I think the annotation should be placed in the interface (`RelOptPlanner`) and not in the implementation class.

Reply:
According to comments, avoid changing the `RelOptPlanner` APIs. so we ignore `@Experimental` annoation.